src/annotator/guest.js

```diff
@@ -535,53 +535,46 @@ export default class Guest extends Delegator {
    * @param {Partial<AnnotationData>} annotation - Initial properties of the
    *   new annotation, other than `target`, `document` and `uri` which are
    *   set by this method.
+   * @return {Promise<AnnotationData>} - The updated annotation with selectors
+   *   and metadata populated.
    */
-  createAnnotation(annotation = {}) {
-    const root = this.element;
-
+  async createAnnotation(annotation = {}) {
```

Comment:
It might be a little more clear to create a new var that gets returned rather than returning the mutated param? 

```
async createAnnotation(partialAnnotation = {}) {
   const annotation = {...partialAnnotation}
```